plugins/out_cloudwatch_logs/cloudwatch_api.c

```diff
@@ -1287,8 +1256,8 @@ int create_log_group(struct flb_cloudwatch *ctx, struct log_stream *stream)
                 flb_sds_destroy(error);
             }
             else {
-                /* error can not be parsed, print raw response to debug */
-                flb_plg_debug(ctx->ins, "Raw response: %s", c->resp.payload);
+                /* error can not be parsed, print raw response */
+                flb_plg_error(ctx->ins, "Raw response: %s", c->resp.payload);
```

Comment:
@matthewfala for same reasoning as my earlier comment on error => warn for raw responses, let's also do warn here?